src/renderer/components/pool/PoolDetails.style.tsx

```diff
@@ -3,16 +3,25 @@ import styled from 'styled-components'
 
 import { media } from '../../helpers/styleHelper'
 
-const ITEM_GAP = '8px'
+export const Container = styled(A.Row)``
 
-export const Container = styled(A.Row)`
-  padding-left: ${ITEM_GAP};
-  ${media.lg`
-    padding: 0;
-  `}
+export const Section = styled(A.Row)`
+  width: 100%;
+  margin-bottom: 10px;
+
+  :nth-last-child(1) {
+    margin-bottom: 0px;
+  }
+
+  overflow-x: scroll;
```

Comment:
This allows just the single section to scroll horizontally, instead of the whole screen scrolling horizontally